images/win/Windows2016-Readme.md

```diff
@@ -2,22 +2,20 @@
 |-|
 | [Windows Server 2022 with Visual Studio 2022 is generally available starting from November, 15](https://github.com/actions/virtual-environments/issues/4488) |
 | [[all OSs] Default Nodejs version will be set to 16 on November, 29](https://github.com/actions/virtual-environments/issues/4446) |
-| [[all OSs] Selenium server version will be upgraded from 3 to 4 on November, 15](https://github.com/actions/virtual-environments/issues/4376) |
-| [[All OSs] Node.js version 10 will be removed from the images on November, 15](https://github.com/actions/virtual-environments/issues/4356) |
 | [Windows-2016 environment will be removed on March 15, 2022](https://github.com/actions/virtual-environments/issues/4312) |
 ***
 # Microsoft Windows Server 2016 Datacenter
 - OS Version: 10.0.14393 Build 4770
-- Image Version: 20211122.1
+- Image Version: 20211229.2
 
 ## Installed Software
 ### Language and Runtime
 - Bash 4.4.23(1)-release
 - Go 1.15.15
 - Julia 1.6.4
-- Kotlin 1.6.0
+- Kotlin 1.5.32
```

Comment:
Version downgrade is related to an API response (https://api.github.com/repos/JetBrains/kotlin/releases/latest) which is used for Kotlin installation. Probably it's expected.

Reply:
This is not expected behaviour that should be fixed in the next images rollout